src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -2009,13 +2009,13 @@ namespace OpenMS
       }
 
       // set precursor information
-      vector<Precursor> precursors;
+      /*vector<Precursor> precursors;
```

Comment:
delete what is currently commented. It's just distracting, including `// set precursor information`.